src/tests/network-no-diff.test

```diff
@@ -36,7 +36,7 @@ post()
     runtime=$(sed -E -n 's/.*@@@ runtime: (.*) @@@.*/\1/p' "$(basename $0).d/baseline.tmux.log")
 
     # If this system can't actually report runtime, bail.
-    if [ "$runtime" -eq "-" ]; then
+    if [ -z "$runtime" -o "$runtime" -eq "-" ]; then
```

Comment:
Use `[ -z "$runtime" ] || [ "$runtime" -eq "-" ]` instead. The `-a` and `-o` operators are less portable and also extend the grammar ambiguously such that they won’t always do what you mean even on systems that support them.
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128_16